src/components/suggest-tip-dialog.tsx

```
@@ -0,0 +1,108 @@
+            <TicketIcon aria-hidden="true" className="size-3.5" />
+            Know something free here?
+          </button>
+        }
+      />
+      <DialogContent>
+        <DialogHeader>
+          <DialogTitle>Suggest a free activity</DialogTitle>
+        </DialogHeader>
+        <form onSubmit={handleSubmit} className="space-y-4">
+          <div className="space-y-2">
+            <Label htmlFor="tip">What's free here?</Label>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)